### PR TITLE
Creating methods jr

### PR DESCRIPTION
**Pull request by Jredish11**
started creating some methods in the stat_tracker class

### CODE REVIEW

**Review by margdaly — APPROVED** [review]

File: spec/stat_tracker_spec.rb [hunk 1]
```diff
@@ -24,9 +24,13 @@
       p @stat_tracker
     end
 
+    it 'processed team data, retrieves data from teams' do
```
Comment (AlejandroLopez1992) [inline spec/stat_tracker_spec.rb:27]: We will need to update this test once with mock data.

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -15,7 +15,24 @@ def self.from_csv(locations)
     end
     new(new_locations)
   end
+  
+  def processed_teams_data(locations)
```
Comment (AlejandroLopez1992) [inline lib/stat_tracker.rb:19]: Good start! Let's keep working on it after a break.

**Review by AlejandroLopez1992 — APPROVED** [review]
Good start, we can review if we can edit this code to become functional.

File: lib/stat_tracker.rb (hunk 2 above)
Comment (margdaly) [reply lib/stat_tracker.rb:19]: After dinner we'll be better!

File: spec/stat_tracker_spec.rb (hunk 1 above)
Comment (margdaly) [reply spec/stat_tracker_spec.rb:27]: - [x]

**Pull request merged**